karate-netty/README.md

```diff
@@ -322,6 +322,13 @@ java -jar karate.jar -T 5 -C src/features
 #### Debug Server
 The `-d` or `--debug` option will start a debug server. See the [Debug Server wiki](https://github.com/karatelabs/karate/wiki/Debug-Server) for more details.
 
+#### Keep Original Headers
+By default, Karate mock server converts all response headers to lowercase according to HTTP/2 standard. This option allows you to retain the original header format, enabling integration with legacy systems or HTTP/1.
+
+```
+java -jar karate.jar --keep-original-headers
```

Comment:
if this is for the mock server, probably needs the `-m` options and / or put this section just under the mocks section ?